core/src/main/com/sinch/sdk/core/utils/databind/Mapper.java

```diff
@@ -28,6 +31,55 @@
 
 public class Mapper {
 
+  public static final PropertyFilter uninitializedFilter =
+      new SimpleBeanPropertyFilter() {
+        @Override
+        public void serializeAsField(
+            Object pojo, JsonGenerator jgen, SerializerProvider provider, PropertyWriter writer)
+            throws Exception {
+
+          if (include(writer)) {
+
+            AnnotatedMember member = writer.getMember();
+
+            // Do not serialize uninitialized fields to avoid sending a null value when not required
+            boolean serialize;
+
+            // Call getter to obtain value: if of OptionalValue type: use it to check serialization
+            // state
+            Object value = pojo.getClass().getMethod(member.getName()).invoke(pojo);
+            if (value instanceof OptionalValue) {
+              serialize = ((OptionalValue<?>) value).isPresent();
+            } else {
+              /* @Deprecated
+               * Not find the expected OptionalValue: fallback to xxxDefined function call
+               * The xxDefined function feature will have to be deprecated in favour of OptionalValue usage
+               * This part could be suppressed as soon as all domains had moved to the OptionalValue support from OAS generated files
+               */
+              serialize =
+                  (boolean) pojo.getClass().getMethod(member.getName() + "Defined").invoke(pojo);
+            }
+            // not set but is field is required ?
+            if (!serialize) {
+              if (member.hasAnnotation(JsonInclude.class)) {
+                JsonInclude annotation = member.getAnnotation(JsonInclude.class);
+                // property is required but was not defined: throw an exception from client side
+                // before
```

Comment:
Check comments formatting for this file (line 49 too)

Reply:
Done for this one.
About line 49: default formatter (spotless+googlecodeformat) setting is line long set to 100 (https://google.github.io/styleguide/javaguide.html#s4.4-column-limit); so it is a "normal" formatting